cmd/go-ycsb/client.go

```diff
@@ -175,7 +175,8 @@ func runClientCommandFunc(cmd *cobra.Command, args []string, doTransactions bool
 		}
 		globalProps.Set(prop.DoTransactions, doTransFlag)
 
-		if threadsArg > 0 {
+		if threadsArg > 1 {
```

Comment:
I prefer using 100 or 200 as the default

Reply:
The default threadcount of the origin YCSB is 1. And the real problem is `threadArg > 0` always true, so it always overrides the property.

Reply:
the default property is 100, so if we really want to pass 1 in threads flag, it can't work now.